content/departments/support/tools/index.md

```diff
@@ -41,26 +41,44 @@ Read more about working with managed instances [here](../process/support-managed
 
 There are some test environments created specifically for the CS team to run tests on when helping customers. They allow the team to reproduce and debug issues without affecting our production instances.
 
+#### Sourcegraph Instances
+
+Add a maintenance [notice](https://docs.sourcegraph.com/admin/config/settings#notices) to the Global settings if you are testing, experimenting or upgrading [cse-k8s](https://cse-k8s.sgdev.org) or [cse-aws-test](https://cse-aws-test.sgdev.org).
```

Comment:
We may want to amend the text of the notice such that it doesn't explicitly say the instance is down. The AER testing could also note what they're doing as far as testing in the banner.